format/Message.fbs

```diff
@@ -0,0 +1,113 @@
+namespace arrow.flatbuf;
+
+table Tuple {
```

Comment:
Tuple isn't defined in Layout.md?  Is this meant to be a fixed width collection/list?